docs/user_guide/metrics.md

```diff
@@ -133,36 +133,40 @@ If building Triton locally, the `TRITON_ENABLE_METRICS_CPU` CMake build flag can
 
 ## Response Cache Metrics
 
-Compute latency metrics in the [Inference Request Metrics table](#inference-request-metrics) above are calculated for the
-time spent in model inference backends. If the response cache is enabled for a
-given model (see [Response Cache](response_cache.md)
+Cache metrics can be reported in two ways:
+
+1. A base set of cache metrics will be reported
+by Triton directly, such as the cache hit/miss counts and durations described
+below.
+
+2. As of 23.02, additional cache metrics may be reported depending on the 
```

Comment:
Do you need the 23.02?

Reply:
I wanted to make it clear to users when the shift from old to new behavior happened. If you think it reads poorly, I can remove it - but I think it's something I've seen in some github issues a few times where users were confused when exactly certain features were added or removed

Reply:
I think it is good practice to add the version number with new features. I have seen similar pattern in Python documentation. https://docs.python.org/3/library/functions.html#aiter

Reply:
Interesting, thank you both for sharing. Maybe we should make this more ubiqutious then.